### PR TITLE
Remove ref name in PR commits page

### PR DESCRIPTION
**Pull request by yp05327**
The branch name display here is based on the repo's default branch which is not correct.
image

For example, if I changed the default branch, the branch name here will also be changed:
image
image
you can confirm this in :
https://try.gitea.io/yp05327/testrepo/pulls/1/commits

I think we do not need to display branch name here, as we already have the branch info above.
image

### CODE REVIEW

**Comment by lunny** [comment]
Why not just remove it from template?

**Comment by yp05327** [comment]
> Why not just remove it from template?

It uses `commits_table.tmpl` which is a shared template.

**Comment by lunny** [comment]
I don't think this is the right way to fix that problem.

**Comment by yp05327** [comment]
Checked all usages, it seems that we can remove it from template safely.

File: routers/web/web.go [hunk 1]
```diff
@@ -1280,7 +1280,7 @@ func registerRoutes(m *web.Route) {
 			m.Get("", repo.SetWhitespaceBehavior, repo.GetPullDiffStats, repo.ViewIssue)
 			m.Get(".diff", repo.DownloadPullDiff)
 			m.Get(".patch", repo.DownloadPullPatch)
-			m.Get("/commits", context.RepoRef(), repo.SetWhitespaceBehavior, repo.GetPullDiffStats, repo.ViewPullCommits)
```
Comment (lunny) [inline routers/web/web.go]: Does this really need to be removed?

Comment (yp05327) [reply routers/web/web.go]: I think this can be removed here, but if you think this is not safe, we can keep it.

**Comment by lunny** [comment]
Since the commit table is shared by some pages, could you also have screenshots for other commits pages?

**Comment by yp05327** [comment]
commits_table.tmpl is used in 3 places. 
image


Repo Commit Page:
Before:
image
After:
image

New PR page: no changes
Before:
image
After:
image

**Review by lunny — APPROVED** [review]

**Review by puni9869 — APPROVED** [review]

**Pull request merged**